bentoml/_internal/bento/build_config.py

```diff
@@ -156,9 +274,53 @@ def write_to_bento(self, bento_fs: FS, build_ctx: str):
                 setup_script, bento_fs, docker_folder, "setup_script"
             )
 
+    @staticmethod
+    def validate_bentofile(yaml_content: t.Dict[str, t.Any]) -> None:
+        docker = yaml_content["docker"]
+
+        if "distro" in docker:
+            distro = docker["distro"]
+            if (
```

Comment:
let's remove this part and put it in the framework guide instead.

Note that the package name may not be "scikit-learn", but "scikit-learn==XXXX".